### ISSUE TITLE
Mod updater downloads forge versions of mods on a fabric server if both versions share a curseforge page

### ISSUE THREAD

**Issue by alexmcgowan112** [Osiris-Team/AutoPlug-Client#183]
<details>
  <summary>Show log</summary>
  
```log
[AP][DEBUG][TaskModDownload] Downloading Architectury-[4.6.77.].jar to '/home/opc/autoplug/downloads/Architectury-[4.6.77.].jar' from 'https://edge.forgecdn.net/files/3893/735/architectury-4.6.77-forge.jar'
```
</details>

**Expected**
The fabric version of the mod should be downloaded

**Actual**
The forge version of the mod was downloaded, so the server tried to run without the mod and crashed

**Comment by alexmcgowan112** [comment]
The mod in question is [Architectury API](https://www.curseforge.com/minecraft/mc-mods/architectury-api)

**Comment by Osiris-Team** [comment]
Yeah that could be, since the latest file is taken and not checked if its fabric or not.

**Comment by Osiris-Team** [comment]
Fixed in 6.0.2
